adapters/excon/README.md

```diff
@@ -0,0 +1,42 @@
+# Excon Adapter
+
+The excon adapter is a community-maintained instrumentation adapter for the [excon][excon-home] HTTP library.
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-adapters-excon
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-adapters-excon` to your `Gemfile`.
+
+## Usage
+
+To install the instrumentation adapter, use `call` with the name of the instrumentation.
```

Comment:
I think you mean to say `use` and not `call`. Also, it's worth pointing out that there are two ways that you can "install" instrumentation. One is using a selective install, where you explicitly list the instrumentation, and the other is using installing all available instrumentation. It would look as follows:

```ruby
OpenTelemetry::SDK.configure do |c|
  c.use_all
end
```